cfg/rippled-example.cfg

```diff
@@ -864,19 +864,8 @@
 #       ...
 #
 #   Example:
-#       type=nudb
 #       path=db/shards/nudb
 #
-#   The "type" field must be present and controls the choice of backend:
-#
-#   type = NuDB
-#       NuDB is recommended for shards.
-#
-#   type = RocksDB
-#
-#       The RocksDB backend also provides these optional parameters:
-#
-#       compression         0 for none, 1 for Snappy compression
 #
 #   Required keys:
 #       path                Location to store the database (all types)
```

Comment:
```suggestion
#       path                Location to store the database
```

Reply:
@MarkusTeufelberger nice catch, thank you.